_monitoring-plugins/ad/api.md

```diff
@@ -668,1043 +1929,1016 @@ POST _plugins/_anomaly_detection/detectors/results/_search
-      "period" : {
-        "interval" : 1,
-        "unit" : "Minutes"
-      }
+    "SWD7ihu9TaaW1zKwFZNVNg": {
+      "ad_execute_request_count": 12,
+      "models": [
+        {
+          "detector_id": "Zi5zTXwBwf_U8gjUTfJG",
+          "model_type": "entity",
+          "last_used_time": 1633398375008,
+          "model_id": "Zi5zTXwBwf_U8gjUTfJG_entity_error13",
+          "last_checkpoint_time": 1633392973682,
+          "entity": [
+            {
+              "name": "error_type",
+              "value": "error13"
+            }
+          ]
+        },
+        ...
+      ],
+      "ad_canceled_batch_task_count": 1,
+      "ad_hc_execute_request_count": 0,
+      "ad_hc_execute_failure_count": 0,
+      "model_count": 15,
+      "ad_execute_failure_count": 2,
+      "ad_batch_task_failure_count": 0,
+      "ad_total_batch_task_execution_count": 27,
+      "ad_executing_batch_task_count": 4
     },
-    "schema_version" : 0,
-    "feature_attributes" : [
-      {
-        "feature_id" : "mYccEnIBTXsGi3mvMd8_",
-        "feature_name" : "total_order",
-        "feature_enabled" : true,
-        "aggregation_query" : {
-          "total_order" : {
-            "sum" : {
-              "field" : "value"
+    "TQDUXEzyTJyV0H6_T4hYUw": {
+      "ad_execute_request_count": 0,
+      "models": [
+        {
+          "detector_id": "Zi5zTXwBwf_U8gjUTfJG",
+          "model_type": "entity",
+          "last_used_time": 1633398375004,
+          "model_id": "Zi5zTXwBwf_U8gjUTfJG_entity_error24",
+          "last_checkpoint_time": 1633388177359,
+          "entity": [
+            {
+              "name": "error_type",
+              "value": "error24"
             }
-          }
-        }
-      }
-    ],
-    "last_update_time" : 1589441737319
+          ]
+        },
+        ...
+      ],
+      "ad_canceled_batch_task_count": 0,
+      "ad_hc_execute_request_count": 0,
+      "ad_hc_execute_failure_count": 0,
+      "model_count": 22,
+      "ad_execute_failure_count": 0,
+      "ad_batch_task_failure_count": 0,
+      "ad_total_batch_task_execution_count": 28,
+      "ad_executing_batch_task_count": 3
+    }
   }
 }
 ```
 
+The `model_count` parameter shows the total number of models running on each node’s memory.
+For historical analysis, you see the values for the following fields:
 
-Use `job=true` to get anomaly detection job information.
+- `ad_total_batch_task_execution_count`
+- `ad_executing_batch_task_count`
+- `ad_canceled_batch_task_count`
+- `ad_batch_task_failure_count`
+
+If haven't run any historical analysis, these values show up as 0.
+
+To get all stats for a specific node:
 
 #### Request
 
 ```json
-GET _plugins/_anomaly_detection/detectors/<detectorId>?job=true
+GET _plugins/_anomaly_detection/<nodeId>/stats
+```
+
+To get specific stats for a node:
+
+#### Request
+
+```json
+GET _plugins/_anomaly_detection/<nodeId>/stats/<stat>
+GET _plugins/_anomaly_detection/<nodeId>/stats/ad_execute_request_count
```

Comment:
Seems this line should be a concrete example request. Duplicate with the line above.